src/ensembl/src/shared/components/textarea/Textarea.scss

```diff
@@ -0,0 +1,17 @@
+.textarea {
+  display: inline-block;
+  width: 100%;
+  border: none;
+  padding: 7px;
+  font-size: 16px;
+  border-radius: 2px;
```

Comment:
The input does not have border-radius by default. Should textarea?

Reply:
No, it shouldn't. Andrea is not convinced with that so I've removed it.